modules/gateway/rpc.go

```diff
@@ -61,7 +61,7 @@ func (g *Gateway) RPC(addr modules.NetAddress, name string, fn modules.RPCFunc)
 func (g *Gateway) RegisterRPC(name string, fn modules.RPCFunc) {
 	id := g.mu.Lock()
 	defer g.mu.Unlock(id)
-	if build.DEBUG {
+	if build.DEBUG && build.Release != "testing" {
```

Comment:
sometimes it's convenient to overwrite an RPC for mocking purposes. I don't see any tests in this PR that do that, though